### DISCUSSION TITLE
Is There any way to use latest Web3Modal wallet connect 3.1 with standalone javascript

### DISCUSSION THREAD

**Discussion by oyin25** [WalletConnect/walletconnect-monorepo#5326] [Developer Support]
I have a java script code for wallet connect but due to the old library of Web3Modal2.6.2, i couldnt perform some action from wagmi command, like getBalance or FetchBalance, can anyone help in that.

the old wallet connect 2.6.2
![Screenshot 2024-09-03 224417](https://github.com/user-attachments/assets/d577ca48-daf1-403d-b43c-9abd108b916f)

the new wallet connect version
![Screenshot 2024-09-03 164038](https://github.com/user-attachments/assets/0ab971e6-61c3-4fc0-8149-6609dffdf267)

**Comment by rtomas** [comment]
Why don't you want to update to latest ? 
Did you try updating wagmi library ?

**Reply by oyin25** [reply]
I would love to use the latest version but I want it to be browser compatible if possible because I just want it to be coded with static web page html and javascript to be precise.
